rocAL/rocAL/source/loaders/audio/audio_loader_sharded.cpp

```diff
@@ -137,7 +137,6 @@ size_t AudioLoaderSharded::remaining_count() {
 void AudioLoaderSharded::reset() {
     for(auto& loader: _loaders) {
         loader->reset();
-        // loader->last_batch_padded_size();
```

Comment:
Is this deletion intended?

Reply:
yes, it was a commented statement. Hence del

Reply:
need to del from prev PRs as well